runtime/basilisk/src/xcm.rs

```diff
@@ -136,15 +137,22 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 	type ChannelInfo = ParachainSystem;
 	type VersionWrapper = PolkadotXcm;
-	type ExecuteOverweightOrigin = MajorityTechCommitteeOrRoot;
-	type ControllerOrigin = MajorityTechCommitteeOrRoot;
+	type ExecuteOverweightOrigin = EnsureMajorityTechCommitteeOrRoot;
+	type ControllerOrigin = EnsureMajorityTechCommitteeOrRoot;
 	type ControllerOriginConverter = XcmOriginToCallOrigin;
+	type WeightInfo = ();
 }
 
 impl cumulus_pallet_dmp_queue::Config for Runtime {
 	type Event = Event;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
-	type ExecuteOverweightOrigin = MajorityTechCommitteeOrRoot;
+	type ExecuteOverweightOrigin = EnsureMajorityTechCommitteeOrRoot;
+}
+
+parameter_type_with_key! {
+	pub ParachainMinFee: |_location: MultiLocation| -> Option<u128> {
+		None
```

Comment:
This will lead to `MinXcmFeeNotDefined` [here](https://github.com/open-web3-stack/open-runtime-module-library/blob/37964651f9b00c53dcd05dcbde97898ef576e1fa/xtokens/src/lib.rs#L545). I'm not yet familiar with those recent xtokens configs, but it seems to be for support of the scenario "Transfer tokens issued by parachain B, from parachain A to parachain B, and use relay chain token as fee".
Related PR: https://github.com/open-web3-stack/open-runtime-module-library/pull/753